### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xd009642**
Added some libraries which I've had cause to use in my day job or in the past. Some big tasks here but I don't think any of them are covered by Rust that well from when I last checked (and a recent check of awesome-rust).

### CODE REVIEW

**Comment by xd009642** [comment]
The GIS ones I hadn't checked up on. There is a rust-gdal binding, but it is very incomplete. I'll add a link to it cause it could do with some love. OGC wise I can only find GeoJSON. I'll amend :smile:

File: README.md [hunk 1]
```diff
@@ -39,12 +39,12 @@ See the [contributing guide](CONTRIBUTING.md) for information on what you can do
 * DDS library [wiki](https://en.wikipedia.org/wiki/Data_Distribution_Service)
 
 #### Geospatial Information Systems
-* OGC standards - multiple crates for standards for encoding, sharing or manipulating geospatial data [link](http://www.opengeospatial.org/standards)
-* Low level GIS functions - rust aternative to [GDAL](http://www.gdal.org/)
+* OGC standards - multiple crates for standards for encoding, sharing or manipulating geospatial data [link](http://www.opengeospatial.org/standards). There's already a create for GeoJSON but none of the others appear to have crates.
```
Comment (ErichDonGubler) [inline README.md]: Could you link the crate to GeoJSON? It's this one, right? https://crates.io/crates/geojson

Also, grammar nazism: s/craete/crate. :)

**Comment by ErichDonGubler** [comment]
Here's my thoughts for this particular iteration of this PR. It's looking even better! Most of the concerns I have are simply about giving concrete action items to aspiring readers of the main list. I hope you don't feel like I'm moving the goalpost here -- I certainly don't know very much about the domains you're PRing for, and so I can only offer high-level feedback. If you want to use another (faster) medium for resolving these issues, contact me and let me know -- I definitely want your changes!

**Comment by xd009642** [comment]
I don't mind the feedback, I'll work on it now  :+1:

**Comment by ErichDonGubler** [comment]
Looks fantastic. Thank you! :)

**Pull request merged**